backend/data/dataSources.py

```diff
@@ -1,65 +1,25 @@
 # group names & display order
 DATAGROUPS = {
-    "community": {
-        "name": "Community support",
-        "displayOrder": 0,
-        "colors": [
-            "#ffffe5",
-            "#f7fcb9",
-            "#d9f0a3",
-            "#addd8e",
-            "#78c679",
-            "#41ab5d",
-            "#238443",
-            "#006837",
-            "#004529",
-        ],  # 9-class YlGn (yellow-green, @colorbrewer2)
-    },
-    "covid": {
-        "name": "COVID vulnerability",
-        "displayOrder": 1,
-        "colors": [
-            "#fff7ec",
-            "#fee8c8",
-            "#fdd49e",
-            "#fdbb84",
-            "#fc8d59",
-            "#ef6548",
-            "#d7301f",
-            "#b30000",
-            "#7f0000",
-        ],  # 9-class OrRd (white-orange-red, @colorbrewer2)
-    },
-    "demographics": {
-        "name": "Demographics",
-        "displayOrder": 2,
-        "colors": [
-            "#fff7f3",
-            "#fde0dd",
-            "#fcc5c0",
-            "#fa9fb5",
-            "#f768a1",
-            "#dd3497",
-            "#ae017e",
-            "#7a0177",
-            "#49006a",
-        ],  # 9-class RdPu (white-purple, @colorbrewer2)
-    },
-    "bias": {
-        "name": "Factors affecting data quality",
-        "displayOrder": 3,
-        "colors": [
-            "#f7fcf0",
-            "#e0f3db",
-            "#ccebc5",
-            "#a8ddb5",
-            "#7bccc4",
-            "#4eb3d3",
-            "#2b8cbe",
-            "#0868ac",
-            "#084081",
-        ],  # 9-class GnBu (white-green-blue, @colorbrewer2)
-    },
+    'community':{
```

Comment:
what is the point in this change? I thought using separate lines was meant to be better formatting?

Reply:
Good spot Benji, yes, although I believe it is subjective- I for example find it much easier to scroll and edit this file (which we are doing manually) when there's a load of expanded lists. When they're in one line you scroll past them easier. With a plugin you can see they're actually sort of like quantitative data- color values from one extreme to another (e.g., yellow to red) and so displaying them in a line makes sense to me since I treat them as a range of numbers which I don't necessarily need to see each one clearly because I know it's a gradient... bit rambly but does that make sense?